### ISSUE TITLE
Letting users change their username

### ISSUE THREAD

**Issue by cesar-brandon** [cesar-brandon/job-platform#29]

**Comment by Shubh8899** [comment]
I would like to work on this one.

**Comment by cesar-brandon** [comment]
Of course, you can improve the functionality or user interface as you see fit. I will be attentive to your contributions. Thank you for your interest in working on this topic.
